src/styles/global.css

```diff
@@ -141,13 +146,41 @@
 
 @layer base {
   body {
-    @apply bg-background font-body leading-base text-body;
+    @apply bg-background font-body text-body text-sm lg:text-md;
```

Comment:
I think we can delete the [global styles](https://github.com/ethereum/ethereum-org-website/blob/dev/src/@chakra-ui/styles.ts#L13) from Chakra now.